msu/systems/serialization/metadata_emulator.nut

```diff
@@ -3,15 +3,23 @@
 {
 	Version = null;
 	Data = null;
+	RealMeta = null;
+	Name = null;
+	ModificationDate = null;
+	CreationDate = null;
+	FileName = null;
+
 	constructor()
 	{
 		this.Data = {};
 		this.Version = ::Const.Serialization.Version;
 	}
 
-	function getVersion()
+	// this would ideally be an alternative static constructor
+	function setRealMeta( __realMeta )
```

Comment:
Why is the parameter prefixed with `__`? We don't have a convention for this afaik.

Reply:
I typoed it apparently